### ISSUE TITLE
No more alt + click drag and alt + click resize?

### ISSUE THREAD

**Issue by Joshua-Ashton** [labwc/labwc#147]
I am guessing this got moved to some bind thing but I am not sure how to set it up

**Comment by bi4k8** [comment]
You'll want to add a binding like the following to your config's `<mouse>` section:

```xml
    <context name="Frame">
      <mousebind button="A-Left" action="Press">
        <action name="Move"/>
      </mousebind>
      <mousebind button="A-Right" action="Press">
        <action name="Resize"/>
      </mousebind>
    </context>
```

At some point in the future, the `"Press"` may need to be changed to `"Drag"`; at that point, these bindings will work identically to [how they work in Openbox](http://openbox.org/wiki/Help:Bindings).

**Comment by johanmalm** [comment]
I'll them to the default binds and `docs/rc.xml.all`

**Comment by johanmalm** [comment]
Fixed by 04d69d871
